pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "iso15118"
-version = "0.23.12"
+version = "0.24.0"
```

Comment:
shouldn't the lock file also be updated?

Reply:
thanks, I am on it

Reply:
done